packages/grid/_modules_/grid/components/menu/columnMenu/GridColumnsMenuItem.tsx

```diff
@@ -48,8 +48,9 @@ GridColumnsMenuItem.propTypes = {
     filterOperators: PropTypes.arrayOf(
       PropTypes.shape({
         getApplyFilterFn: PropTypes.func.isRequired,
-        InputComponent: PropTypes.elementType,
+        InputComponent: PropTypes.func,
```

Comment:
I think it is because the two JSX elements have different props and then are considered as different objects. So it provides the common property: they are both function